RFS/src/test/java/com/rfs/integration/EndToEndTest.java

```diff
@@ -0,0 +1,129 @@
+package com.rfs.integration;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+
+import com.rfs.framework.ElasticsearchContainer;
+import com.rfs.framework.OpenSearchContainer;
+import com.rfs.framework.OpenSearchContainer.Version;
+import com.rfs.framework.SimpleRestoreFromSnapshot;
+
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Tests focused on setting up whole source clusters, performing a migration, and validation on the target cluster
+ */
+@Slf4j
+public class EndToEndTest {
+
+    protected Object sourceCluster;
+    protected Object targetCluster;
+    protected SimpleRestoreFromSnapshot simpleRfsInstance;
+
+    @ParameterizedTest(name = "Target OpenSearch {0}")
+    @ArgumentsSource(SupportedTargetCluster.class)
+    @Disabled
+    public void migrateFrom_ES_v6_8(final OpenSearchContainer.Version targetVersion) throws Exception {
+        // Setup
```

Comment:
one last comment on building this out.  It would be great if the way to generate this were all through REST commands so that we could spin it around to use for CDC tests too.

Reply:
I think we need to test different things in different systems - happy path we could have good alignment, but edge cases for CDC will need much more flexiblity.  That said - I'll keep in mind good opportunities for reuse, thanks for bring this up.